src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+
 public class FileWork {
+    private static final char SPECIAL_LOWERCASE_W_CHARACTER = 'w';
+
     public String[] readFromFile(String fileName) {
-        return null;
+        StringBuilder allText = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            String line = bufferedReader.readLine();
+            while (line != null) {
+                allText.append(line).append(" ");
+                line = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("ERROR: It seems there is no file: " + fileName, e);
+        }
+        return getWordsWithLowerCaseW(allText);
+    }
+
+    private String[] getWordsWithLowerCaseW(StringBuilder allText) {
+        if (allText == null || allText.toString().isEmpty()) {
```

Comment:
```suggestion
        if (allText.toString().isEmpty()) {
```
you are creating this object, it can be null in your case

Reply:
Thank you!